homepage/homepage/content/docs/server-side/setup.mdx

```diff
@@ -117,3 +117,36 @@ Currently, the Jazz Loader is tested on the following edge environments:
 
 - Edge runtime environment that supports WebAssembly
 - `jazz-tools/load-edge-wasm` must be imported before any Jazz import
+
+
+## Node-API
+
+Jazz uses a WASM-based crypto implementation for maximum compatibility and performance across a wide variety of environments. 
+
```

Comment:
It would be nice to briefly explain what Node-API is, worth also mentioning N-API works also on Deno and Bun